doc/develop/gui.rst

```diff
@@ -0,0 +1,145 @@
+
+*MicroscopyGUIData* corresponds to a specific GUI tab, and contains data related to the shared state of that tab. 
+
+	- **LiveViewGUIData(MicroscopyGUIData):**
+	
+		Represents an interface used to only show the current data from the microscope.
+	
+	- **SparcAcquisitionGUIData(MicroscopyGUIData):**
+	
+		Represents an interface used to select a precise area to scan and
+		acquire signal. It allows fine control of the shape and density of the scan. 		It is specifically made for the SPARC system.
+
+	- **ChamberGUIData(MicroscopyGUIData):**
+	
+		Represents an interface used by the chamber view tab. 
+		
+	- **AnalysisGUIData(MicroscopyGUIData):**
+	
+		Represents an interface used to show the recorded microscope data. Typically
+		it represents all the data present in a specific file. All the streams should be StaticStreams
+		
+	- **ActuatorGUIData(MicroscopyGUIData):**
+	
+		Represents an interface used to move the actuators of a microscope. It might
+		also display one or more views, but it's not required. Typically used for the SECOM and SPARC(v2) alignment tabs.
+		
+	- **SecomAlignGUIData**, **SparcAlignGUIData**, and **Sparc2AlignGUIData**:
+	
+		Represents an interface used for alignment tabs. 
+
+
+Controllers
+===========
+
+.. figure:: gui_tree.*
+    :width: 100 %
+    :align: center
+
+The Odemis GUI has several main controllers that allow a user to directly control functionality. 
+
+The top level controllers include:
+    1. **TabBarCont** (``/gui/cont/tabs.py``)
+    2. **MenuCont** (``/gui/cont/menu.py``)
+    3. **Microscope** (``/gui/cont/microscope.py``)
+    
+|
+1. **TabBarCont:**
+
+	Odemis contains many different tabs in its interface for different modes of operation. Each tab is controlled by a corresponding tab controller, wherein each inheriting from the *Tab* base class. These include:
+	
+	- **SecomStreamTab:**
+	
+		Provides views for streams and corresponding controls for the SECOM and SECOMV2 platform. 
+	
+	- **SparcAcquisitionTab:**
+	
+		Provides views for streams and acquisition controls for the SPARC and SPARCV2 platform. 
+	
+	- **ChamberTab:**
+	
+		Handles control of the measurement chamber state. 
+	
+	- **AnalysisTab:** 
+	
+		Handles the loading and displaying of acquisition files.
+		
+	- **SecomAlignTab:**
+	
+		Tab for the lens alignment on the SECOM and SECOMv2 platform.
+		
+	- **SparcAlignTab:**
+	
+		Tab for the mirror/fiber alignment on the SPARCV1.
+		
+	- **Sparc2AlignTab:**
+	
+		Tab for the mirror/fiber alignment on the SPARCv2. Note that the basic idea is similar to the SPARCv1, but the actual procedure is entirely different.
+   
+
+The following controllers are subcontrollers of a tab controller. 
+
+    1. **StreamCont** (``/gui/cont/streams.py``)
```

Comment:
Please spell out the full class names. `StreamController`, `StreamBarController`